datalab/data/_csv.py

```diff
@@ -176,4 +176,4 @@ def sample_to(self, count, skip_header_rows, strategy, target):
         datalab.utils.gcs_copy_file(f.name, target)
     else:
       with open(target, 'w') as f:
-        df.to_csv(f, header=False, index=False, delimiter=self._delimiter)
+        df.to_csv(f, header=False, index=False, sep=str(','))
```

Comment:
Yes, it should. I changed things up when I was debugging why simply casting to `str` wasn't working (because an import was overriding `str`), and I forgot to change it back.

Thanks for catching that.